azure/template.json

```diff
@@ -92,6 +92,7 @@
         {
             "apiVersion": "2017-05-10",
             "name": "app-service-certificate",
+            "resourceGroup": "[parameters('sharedEnvResourceGroup')]",
```

Comment:
```suggestion
            "resourceGroup": "[parameters('sharedManagementResourceGroup')]",
```

Reply:
No we are deploying to the shared env resource group from the keyvault in the management group.  matches this example https://github.com/SkillsFundingAgency/das-login-service/blob/master/azure/template.json#L70